raiden/tests/utils/network.py

```diff
@@ -133,36 +133,39 @@ def payment_channel_open_and_deposit(
         partner=app1.raiden.address, settle_timeout=settle_timeout, given_block_identifier="latest"
     )
     assert channel_identifier
-    canonical_identifier = CanonicalIdentifier(
-        chain_identifier=state_from_raiden(app0.raiden).chain_id,
-        token_network_address=token_network_proxy.address,
-        channel_identifier=channel_identifier,
-    )
 
-    for app in [app0, app1]:
-        # Use each app's own chain because of the private key / local signing
-        token = app.raiden.chain.token(token_address)
-        payment_channel_proxy = app.raiden.chain.payment_channel(
-            canonical_identifier=canonical_identifier
+    if deposit != 0:
```

Comment:
A value `0` is invalid. Please raise an exception instead.

Reply:
as discussed since this is in test utils its fine